### PR TITLE
Remove android info and add limitations

### PR DESCRIPTION
**Pull request by leighajarett**
_Description of what this PR is changing or adding, and why:_
Removing the bit about Android and adding in some limitations mentioned by @cyanglaz 

Closes https://github.com/flutter/website/issues/8741

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/development/platform-integration/ios/platform-views.md [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+which means that the native
+`UIView` is appended to the view hierarchy.
+
+To create a platform view on iOS,
+use the following instructions:
+
+### On the Dart side
+
+On the Dart side, create a `Widget`
+and add the following build implementation,
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:38]: Don't need "following" twice in the same sentence.

```suggestion
and add the build implementation,
```

File: src/development/platform-integration/ios/platform-views.md [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+as shown in the following steps.
+
+In your Dart file, for example
+do the following in `native_view_example.dart`:
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:42]:
```suggestion
In the Dart widget file, make changes similar to those
show in `native_view_example.dart`:
```

File: src/development/platform-integration/ios/platform-views.md [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+}
+```
+</li>
+
+<li markdown="1">Implement a `build()` method:
+
+<?code-excerpt "lib/platform_views/native_view_example_3.dart (iOSCompositionWidget)"?>
+```dart
+Widget build(BuildContext context) {
+  // This is used in the platform side to register the view.
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:98]:
```suggestion
  // Use this on the platform side to register the view.
```

File: src/development/platform-integration/ios/platform-views.md [hunk 4]
```diff
@@ -0,0 +1,434 @@
...
+</ol>
+
+For more information, see the API docs for:
+[`UIKitView`][].
+
+[`UIKitView`]: {{site.api}}/flutter/widgets/UiKitView-class.html
+
+### On the platform side
+
+On the platform side, you use the either Swift or Objective-C:
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:121]:
```suggestion
On the platform side  use either Swift or Objective-C:
```

File: src/development/platform-integration/ios/platform-views.md [hunk 5]
```diff
@@ -0,0 +1,434 @@
...
+
+{% samplecode ios-platform-views %}
+{% sample Swift %}
+
+Implement the factory and the platform view.
+The `FLNativeViewFactory` creates the platform view,
+and the platform view provides a reference to the `UIView`.
+For example, `FLNativeView.swift`:
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:129]:
```suggestion
For example, as shown in `FLNativeView.swift`:
```

File: src/development/platform-integration/ios/platform-views.md [hunk 6]
```diff
@@ -0,0 +1,434 @@
...
+    public static func register(with registrar: FlutterPluginRegistrar) {
+        let factory = FLNativeViewFactory(messenger: registrar.messenger)
+        registrar.register(factory, withId: "<platform-view-type>")
+    }
+}
+```
+
+{% sample Objective-C %}
+
+Add the headers for the factory and the platform view.
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:234]:
```suggestion
In Objective-C, add the headers for the factory and the platform view.
```

File: src/development/platform-integration/ios/platform-views.md [hunk 7]
```diff
@@ -0,0 +1,434 @@
...
+
+[`FlutterPlatformView`]: {{site.api}}/objcdoc/Protocols/FlutterPlatformView.html
+[`FlutterPlatformViewFactory`]: {{site.api}}/objcdoc/Protocols/FlutterPlatformViewFactory.html
+[`PlatformView`]: {{site.api}}/javadoc/io/flutter/plugin/platform/PlatformView.html
+
+## Putting it together
+
+When implementing the `build()` method in Dart,
+you can use [`defaultTargetPlatform`][]
+to detect the platform, and decide what widget to use:
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:376]:
```suggestion
to detect the platform, and decide which widget to use:
```

File: src/development/platform-integration/ios/platform-views.md [hunk 8]
```diff
@@ -0,0 +1,434 @@
...
+
+<?code-excerpt "lib/platform_views/native_view_example_3.dart (TogetherWidget)"?>
+```dart
+Widget build(BuildContext context) {
+  // This is used in the platform side to register the view.
```
Comment (sfshaza2) [inline src/development/platform-integration/ios/platform-views.md:381]:
```suggestion
  // Use this on the platform side to register the view.
```

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Hi, @leighajarett! I did much of the review on this page before noticing that you made this PR under our old information architecture. The new location for this file is: /src/platform-integration/ios/platform-views.md.

Can you make the edits that I've provided and also merge with the current repo so that it's in the correct place?

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Hi, @leighajarett! I did much of the review on this page before noticing that you made this PR under our old information architecture. The new location for this file is: /src/platform-integration/ios/platform-views.md.

Can you make the edits that I've provided and also merge with the current repo so that it's in the correct place?

File: src/development/platform-integration/ios/platform-views.md [hunk 9]
```diff
@@ -0,0 +1,434 @@
...
+  within a Flutter app.
+  If you'd like to embed native Android views
+  in your Flutter app,
+  see [Hosting native Android views][].
+{{site.alert.end}}
+
+[Hosting native Android views]: {{site.url}}/development/platform-integration/android/platform-views
+
+
+iOS only uses Hybrid composition,
```
Comment (domesticmouse) [inline src/development/platform-integration/ios/platform-views.md:28]: suggestion: "iOS uses only hybrid composition"

Also, hybrid composition doesn't appear to be defined on this page before this mention. Can we link to the definition, or should we lose the use of this undefined term?

File: src/development/platform-integration/ios/platform-views.md [hunk 10]
```diff
@@ -0,0 +1,434 @@
...
+This allows Flutter apps to be fast, 
+as the main platform thread is rarely blocked.
+
+When a platform view is rendered with hybrid composition, 
+the Flutter UI is composed from the platform thread. 
+The platform thread competes with other tasks 
+like handling OS or plugin messages.
+
+When an iOS PlatformView is on screen, the screen refresh rate is 
+capped at 80fps to avoid rendering janks.
```
Comment (domesticmouse) [inline src/development/platform-integration/ios/platform-views.md:411]: I'm curious why 80fps was chosen. 60fps and 90fps seem sane, but 80?

**Review by domesticmouse — APPROVED** [review]

**Comment by leighajarett** [comment]
This got messy with moving to new information architecture so moving to a new PR

**Pull request closed without merging**